sdk/subscription/Microsoft.Azure.Management.Subscription/src/Microsoft.Azure.Management.Subscription.csproj

```diff
@@ -1,29 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nothing in this file should be changed except the version. And you also need to bump the major version in AssemblyInfo.cs because there are many breaking changes. And please also update AzSdk.RP.props

Reply:
Hi @Ochirkhuyag, you also need to do the following things:
1. Update the breaking changes description in this file
2. Also bump version in AssemblyInfo.cs
3. Update AzSdk.RP.props